ansible/config_sonic_basedon_testbed.yml

```diff
@@ -702,6 +702,16 @@
         shell: bash "/etc/sonic/apply_clet.sh"
         when: stat_result.stat.exists is defined and stat_result.stat.exists
 
+      - block:
```

Comment:
Hi @nnelluri-cisco , can we move the logic to golden config? This is more like our PROD wokflow: https://github.com/sonic-net/sonic-mgmt/pull/16033
One thing to remind is that the golden config override will replace table fully. So we need to create table with full entry.

We also have YANG validation for  golden config override which is preferred.